### DISCUSSION TITLE
Footnotes are not grouped when using remark-footnotes

### DISCUSSION THREAD

**Discussion by anhari** [remarkjs/remark#612] [Q&A] [answered]
Is there a way to group footnotes under some HTML element so that I can style their container?

Perhaps I'm using the plugin wrong:

``` javascript
import ReactMarkdown from "react-markdown";
import footnotes from "remark-footnotes";

function footnoteReference(props) {
  return (
    <sup id={"ref-" + props.identifier}>
      <a href={"#def-" + props.identifier}>{props.label}</a>
    </sup>
  );
}

function footnoteDefinition(props) {
  return (
    <div className="flex footnotes first:mt-1" id={"def-" + props.identifier}>
      <div className="flex items-baseline space-x-4">
        <span>{props.label}.</span> <span>{props.children}</span>
        <a href={"#ref-" + props.identifier}>↩︎</a>
      </div>
    </div>
  );
}

const renderers = {
  footnoteReference,
  footnoteDefinition,
};

function Article() {
  return (
     <ReactMarkdown children={body} renderers={renderers} plugins={[footnotes]} />
  )
}
```

Article with two footnotes creates two siblings with no parent:

<img width="460" alt="image" src="https://user-images.githubusercontent.com/6628875/105272756-f976be80-5b67-11eb-8178-cdf443ae00fd.png">

**Comment by ChristianMurphy** [comment, marked as answer]
The example in the readme groups footnotes into a `div` https://github.com/remarkjs/remark-footnotes
Some options:
1. use the utilities directly, as shown in https://github.com/remarkjs/remark-footnotes
2. wait a bit, `react-remark` is considering leveraging rehype which would allow the example in remark-footnotes to work more directly https://github.com/remarkjs/react-markdown/issues/427
3. create a plugin, footnotes can be moved any where you prefer using a plugin/transform (guide for creating plugins https://unifiedjs.com/learn/, and plugin usage with `react-markdown` here: https://github.com/remarkjs/react-markdown#use-a-plugin)
4. use https://github.com/remarkjs/react-remark which also is remark + rehype based and should be able to run the example in https://github.com/remarkjs/remark-footnotes
